raster/src/main/scala/geotrellis/raster/GridBounds.scala

```diff
@@ -76,11 +76,7 @@ case class GridBounds(colMin: Int, rowMin: Int, colMax: Int, rowMax: Int) {
   def width = colMax - colMin + 1
   def height = rowMax - rowMin + 1
 
-  @deprecated("This will return a `Long` in 2.0. Until then, sizeLong may be more accurate.", "1.2")
-  def size: Int = width * height
-
-  // TODO Mark for deprecation in 3.0 when 2.0 comes out!
-  def sizeLong: Long = width.toLong * height.toLong
```

Comment:
Is it okay to remove `sizeLong` without deprecating first?

Reply:
Not at all